### ISSUE TITLE
Comment if you want to join the iOS beta 2.17.4 via Testflight – and then please open issues for any problems you encounter :)

### ISSUE THREAD

**Issue by marinofaggiana** [nextcloud/ios#260]
The new beta for Nextcloud iOS 2.17.4 is open, write here your **email Apple** if you want receive the invite for TestFlight :

https://itunes.apple.com/it/app/testflight/id899247664?mt=8

This is only for new users.

Do not write here a issue, open a new ticket.

End porting migrate Database, test : ... All !! 

**(probable several bug and crash, sorry this is alpha version)**

- Improvement Auto Upload 
- New View for Shares on menu "More"

Thanks.

**Comment by Dirk23** [comment]
dirk@diehildebrands.de

**Comment by nbittmann** [comment]
Can you add nils.bittmann@googlemail.com

**Comment by wallenium** [comment]
No problems until now. Working well. Good job.

Short question: is there a list of files which are currently uploading? Or just the number?

**Comment by rakekniven** [comment]
Hello @wallenium : See "more" menu and select "Transfer". You get upload list with details and options to stop/restart/remove uploads.

**Comment by rakekniven** [comment]
@marinofaggiana Everything good so far. Using all Testflight versions and really like the new share option.

**Comment by khlschrnk** [comment]
Please add me with respect to #266: leo.goslich@email.de

**Comment by marinofaggiana** [comment]
Thanks @rakekniven !!

**Comment by StoreEverything** [comment]
@marinofaggiana I've sent you a pm with the email address

**Comment by StoreEverything** [comment]
Ok, previous files that wouldn't upload now appear to be uploading.

I'm getting an error "WebDAV Locked: The resource that is being accessed is locked"

Saw an error "100" and a "-1"

**Comment by marinofaggiana** [comment]
Locked for example if you change Ip in upload (3G - WiFi) ... the system automatic retry .

**Comment by sergey-nb** [comment]
ibara@yandex.ru

**Comment by mrnoisytiger** [comment]
Can you add xoxorockoutloud123@gmail.com?

**Comment by marinofaggiana** [comment]
Please tester, make test on Auto Upload Photos / Videos, thanks.

**Comment by StoreEverything** [comment]
How long do you think until this release goes public? Got two other phones (family) with same problems I had.

**Comment by sergey-nb** [comment]
1. When I logged into the program, I made a mistake in the server address, after correcting the address to the correct one, it is not necessary to press the Login button three times. That is how it should be? (I use self-signed certificate, NextCloud for IOS 2.17.4.00014, Nextcloud Server 12.0.0)
2. How can I set up folders for automatic file's synchronization everytime with offline file access in this folder? How can I distinguish folders where all files are synchronized and accessible locally, and folders where it is not so.

**Comment by sergey-nb** [comment]
Sorry.
In my 1. question "....it is necessary to press the Login button three times....". Only after 3 times press "Login" I was able to enter the program.

**Comment by marinofaggiana** [comment]
2. Use favorite

**Comment by binarybucks** [comment]
Please add mail@alr.st

**Comment by sergey-nb** [comment]
Favorites folders does not automatically download files. Just a yellow star appears and that's it. I made folder favorites, turned off the network and none of the 100 files were available offline

**Comment by marinofaggiana** [comment]
@sergey-nb try with new version (15)

**Comment by d-jahren** [comment]
I have the same issue as StoreEverything. Should I PM you with email for the beta or wait for the public release?

**Comment by tzapu** [comment]
hi, @marinofaggiana could you add alex@tzapu.com as well? thanks

**Comment by emresaracoglu** [comment]
emsaracoglu@gmail.com

_Also, Could you send me an email? I want to talk to you about another project_

**Comment by simonspoerri** [comment]
Can you add simon.spoerri@styleskill.com? Thank you!

**Comment by marinofaggiana** [comment]
ping ... Hi tester ! who has a very large number photos / videos in your iPhone Camera Roll ??

**Comment by tzapu** [comment]
@marinofaggiana got about 6000, just finally managed to download all from icloud (originals)

enough ?

cheers

**Comment by marinofaggiana** [comment]
@tzapu yes, if possible, a video capture when the app go back on foreground and detect (AutoUpload) a 10 new photo ... I want see the time of detect ... Thanks

**Comment by tzapu** [comment]
ok, will work out how to do it and upload

**Comment by tzapu** [comment]
is this ok?
https://www.dropbox.com/s/jbuz6c6w2mgnqjm/new-photos.mov?dl=0
seems really fast. 
iphone camera roll was about 6000, made 10 new pics after turning upload on. so next cloud had 0 photos before, 10 after.

**Comment by marinofaggiana** [comment]
oh! Thanks !! ipHone model ? 6 6s 7 ??

**Comment by tzapu** [comment]
iphone 6s
this beta really feels fast for me. the stable version locks up all the time. have not noticed it once since on beta.

good job

**Comment by robvh84** [comment]
Please add me to the beta group: rob@robvanhamersveld.nl

**Comment by rakekniven** [comment]
Hello @tzapu Thanks for your ongoing testing help.
btw. No need to use dropbox. You have your own cloud *SCNR*

**Comment by tzapu** [comment]
thanks for the hint @rakekniven . it did not occur to me actually to use it.
i am not using nextcloud as main repo yet due to some other issue i have with it changing photos dates (reported here somewhere) and it's  just simpler to use dropbox because of the 'move to dropbox' contextual command
https://www.dropbox.com/s/6011l0be85i23mq/Screenshot%202017-07-02%2014.08.16.png?dl=0
also, i just did it again for screenshots. it s just simpler. maybe these two things could be added in time to nextcloud? it would be perfect :D

**Comment by himbeles** [comment]
Could you please add me to TestFlight: lri@me.com?

**Comment by michaelnew** [comment]
Hey I'd love to do some testing as well. Email is mikenew12@gmail.com

**Comment by herohamp** [comment]
christinafmoore@yahoo.com

**Comment by herohamp** [comment]
I would love to do some testing! christie@bugrealty.com

**Comment by serenity-networks** [comment]
I need this BADLY AS SOON AS POSSIBLE for iTunes account carmenhammond@icloud.com   AND mike@serenity-networks.com   We have a MAJOR deployment being held up and I'm confident this will fix everything.

**Comment by herohamp** [comment]
Sorry somehow I lost access to the christie@bugrealty.com email, can It be shared to me at hampmoe@icloud.com 😭

**Comment by brunkj** [comment]
still looking for people for testflight?  jason.brunk@gmail.com :)

**Comment by marvinpeng** [comment]
I have upload issue, please add me (marvin.peng@me.com) to test beta version. Thanks.

**Comment by marinofaggiana** [comment]
Please write issue for bug or crash. Tanks.

**Comment by heichel** [comment]
Would love to test!

lukas19096@t.lhm.onl

**Comment by Kumapapa2012** [comment]
Please invite me to Test Flight.
kumapapa2012@ezweb.ne.jp

**Comment by marinofaggiana** [comment]
TestFlight CLOSED.

Thanks a lot for your tests.

**Comment by murder666** [comment]
When 2.17.4 will be available for use in appstore?

**Comment by rakekniven** [comment]
Hello @marinofaggiana Same question from my side: When will our beloved device telling me about the new final release?

**Comment by marinofaggiana** [comment]
Hi dears ! It's in waiting for review on App Store 👍

**Comment by rakekniven** [comment]
Great. Will try have related issues marked for milestone. What is next? v2.17.5?

Great work!!!

**Comment by marinofaggiana** [comment]
Thanks @rakekniven  !! Yes 2.17.5 😋

**Comment by sergey-nb** [comment]
By "long" tap on any file -> program closes without any error message.
(iPhone 6S, IOS 10.3.2, IOS client 2.17.4.00048, NextCloud Server 12.0.0)

**Comment by sergey-nb** [comment]
Clarification: this happens only after uploading the file (after "+"->upload file) from the Documents 6 to root directory. But if you move the file to any directory, "long" tap opens context menu and works fine

**Comment by rakekniven** [comment]
Hello @sergey-nb 
Please file an new issue.
v2.17.4 is final and released at app store.
Thank you.

**Comment by shock143** [comment]
anas@altunji.com
Thanks

**Comment by rakekniven** [comment]
Hello @shock143 
Please see my previous comment and check #321
